### DISCUSSION TITLE
next/image does not display my image

### DISCUSSION THREAD

**Discussion by Salman-Samian** [vercel/next.js#24535] [Help]
### What version of Next.js are you using?

^10.0.0

### What version of Node.js are you using?

v14.5.0

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

npm run dev

### Describe the Bug

I recently change my html <img ... /> tag to (next/image) <Image ... /> as the following code:

```
<Image
                    src="/assets/images/logo.png"
                    alt="Picture of the author"
                    width={140}
                    height={21}
                  />
```

but I did not get any result in my browser either no error, After I right click on my image at the position should be and click (Open image a new tab) just get this result : 

`data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7`

I test with every other image option such as jpeg or png but every time I just get this result.

### Expected Behavior

please help me or fix this issue. 

Thanks.

### To Reproduce

please fix this issue, please.

**Comment by jdulik** [comment]
Any logs in the terminal?

**Reply by Salman-Samian** [reply]
no error log in the terminal either .

**Comment by ndstephens** [comment]
The Image tag worked for me from v10.0.8 on down.  Starting at v10.0.9 it no longer works

**Reply by Salman-Samian** [reply]
Thanks, I will try other versions.
